pyproject.toml

```
@@ -59,7 +59,7 @@ dependencies = [
   "rechunker",
   "scipy",
   "shapely>=2.0",
-  "sparse<=0.14",
+  "sparse<=0.15.1",
```

```suggestion
  "sparse",
```
My bad again. The issue in sparse that triggered the pin has been fixed.